### PR TITLE
Add machine checking to mode influx log

### LINKED ISSUE #48: job_id duplicated in tag and fields of InfluxDB logging
The name `job_id` appears in both the tags and fields of records sent to InfluxDB, but the tag and field do not refer to the same thing. The job_id::tag refers to the harness test ID, while the job_id::field refers to the SLURM job_id.

Background:
A record in InfluxDB is a document (a series of key-value pairs, also called fields). Each record is identified by it's tag (another set of key-value pairs). When querying influxDB, influx typically searches the fields first, then if it doesn't find an entry in the fields, it moves to the tag. So, having a field and a tag named the same but meaning different things is confusing and potentially the cause of issues.

Proposed fix:
Rename job_id::tag to test_id::tag (which duplicates test_id::field, but these refer to the same thing).

### LINKED ISSUE #55: Add checking of RGT_MACHINE_NAME to influx_log mode
Feature request -- under the influx_log mode, app/test paths are searched for existing tests, and any completed tests are logged under the current machine's name. So if Spock results are in the same directory as Crusher results, but you run the influx_log command from Crusher, all results are labeled as Crusher.

The influx_log mode already parses the test's status files, so adding this check should be possible.

I propose **checking if RGT_MACHINE_NAME is equal to the rgt_machine_name from the status files**.

### PR DESCRIPTION
**Pull request by hagertnl**
This is a patch for checking the machine name for a test instance before logging it under influx_log mode

### CODE REVIEW

**Comment by hagertnl** [comment]
This PR also implements the unique tags and fields for InfluxDB, and pulls them out to a class variable for reusability

File: harness/libraries/status_file.py [hunk 1]
```diff
@@ -851,6 +895,8 @@ def get_status_info(test_id, event_type, event_subtype,
         test_instance_info['test'],
         test_instance_info['test_id'], apptest_layout.test_build_dirname)
 
+    test_instance_info['machine'] = os.environ['RGT_MACHINE_NAME'] if 'RGT_MACHINE_NAME' in os.environ else no_value
```
Comment (verolero86) [inline harness/libraries/status_file.py:899]: @hagertnl - what happens when we pass `no_value`? are things still findable in influx through some other mechanism?

Comment (hagertnl) [reply harness/libraries/status_file.py:899]: The record would still be findable, there would actually be another machine listed as `[NO_VALUE]` in the Influx DB at that point. Then we could potentially use an update script to update record's tags. There shouldn't ever be a situation where RGT_MACHINE_NAME isn't set for a harness run, right?

**Comment by hagertnl** [comment]
One pending item I have uncovered in developing utility scripts is that the user is no longer being logged to Influx. This should be remedied before merging. Will commit this afternoon.

File: harness/libraries/status_file.py (hunk 1 above)
Comment (verolero86) [reply harness/libraries/status_file.py:899]: in theory, no. in practice... i hope not 😄

**Review by verolero86 — APPROVED** [review]

**Pull request merged**